src/test-helpers.ts

```diff
@@ -8,9 +8,13 @@ export const sleep = (milliseconds: number) => {
   return new Promise(resolve => setTimeout(resolve, milliseconds));
 };
 
-export const setupStore = () => {
+export const setupStore = (initialState?: any) => {
   const sagaMiddleware = createSagaMiddleware();
-  const store = createStore(reducer, applyMiddleware(sagaMiddleware));
+  const store = createStore(
+    combineReducers({ resources: reducer }),
```

Comment:
This is a good point we should aggregate the resources logic in one particular place.